.github/workflows/ci.yaml

```
@@ -27,27 +31,30 @@ jobs:
       - name: Active CI cargo config
         run: mv .cargo/config_ci.toml .cargo/config.toml
 
-      - name: Install cargo binstall
-        uses: cargo-bins/cargo-binstall@main
-
       - name: Install nextest
-        run: cargo binstall cargo-nextest --secure --force
+        uses: taiki-e/install-action@nextest
 
       - name: Install dependencies
         run: sudo apt-get update; sudo apt-get install --no-install-recommends libasound2-dev libudev-dev libwayland-dev libxkbcommon-dev
 
+      - name: Set LD_LIBRARY_PATH
+        id: ld_library_path
+        run: |
+          # Setting LD_LIBRARY_PATH is a workaround for <https://github.com/TheBevyFlock/bevy_new_2d/pull/318#issuecomment-2585935350>.
+          echo "LD_LIBRARY_PATH=$(rustc --print target-libdir)" >> $GITHUB_ENV
```

```suggestion
        run: |
          # Setting LD_LIBRARY_PATH is a workaround for <https://github.com/TheBevyFlock/bevy_new_2d/pull/318#issuecomment-2585935350>.
          echo "LD_LIBRARY_PATH=$(rustc --print target-libdir)" >>"${GITHUB_ENV}"
```
I'm still curious what effect this has on CI time if any, but happy to merge it for now after this nit.